tests/integration/static_ip_address_test.py

```diff
@@ -918,6 +922,76 @@ def test_merge_ip_enabled_property_from_current(setup_eth1_static_ip):
     assertlib.assert_state_match(desired_state)
 
 
+@pytest.mark.skipif(
+    nm_minor_version() < 54,
+    reason="IPv4 forwarding is not supported on NM 1.54-",
+)
+def test_ipv4_sysctl_forwarding(setup_eth1_static_ip):
+    desired_state = {
+        Interface.KEY: [
+            {
+                Interface.NAME: "eth1",
+                Interface.IPV4: {
+                    InterfaceIPv4.ENABLED: True,
+                    InterfaceIPv4.ADDRESS: [
+                        {
+                            InterfaceIPv4.ADDRESS_IP: IPV4_ADDRESS1,
+                            InterfaceIPv4.ADDRESS_PREFIX_LENGTH: 24,
+                        }
+                    ],
+                    InterfaceIPv4.FORWARDING: True,
+                },
+            }
+        ]
+    }
+    apply_with_description(
+        "Enable IPv4 forwarding on eth1",
+        desired_state,
+    )
+    assertlib.assert_state_match(desired_state)
+    desired_state[Interface.KEY][0][Interface.IPV4][
+        InterfaceIPv4.FORWARDING
+    ] = False
+    apply_with_description(
+        "Disable IPv4 forwarding on eth1",
+        desired_state,
+    )
+    assertlib.assert_state_match(desired_state)
+
+
+@pytest.mark.skipif(
+    nm_minor_version() >= 54,
+    reason="IPv4 forwarding is not supported on NM 1.54-",
+)
+def test_ipv4_sysctl_forwarding_unsupported_error(setup_eth1_static_ip):
```

Comment:
The condition says `>=54`, but the reason says `not supported.


Please also add test case for applying current forwarding value to old NM and nmstate still pass.